packages/ticker/package.json

```diff
@@ -23,6 +23,9 @@
     "lib",
     "dist"
   ],
+  "peerDependencies": {
+    "@pixi/app": "5.2.1"
```

Comment:
Again, we need a typing solution to decouple. peerDependencies isn't the correct solution here.

Reply:
go away from the nintendo of my son app